### PR TITLE
Include JEP into save-demo-cpg image

### PR DESCRIPTION
**Pull request by petertrr**
* Custom builder with `gcc` and python headers pre-installed
* `requirements.txt`
* Some values are  hardcoded in respect with Python 3.10

TODOs: un-hardcode python version from paths, optimize image size

### CODE REVIEW

**Comment by petertrr** [comment]
JEP is working, but something is not right on Python side:
```
13:18:17.793 [Thread-5] INFO d.f.aisec.cpg.TranslationManager - Parsing /tmp/python401744558970441993/demo.py
13:18:17.796 [Thread-5] INFO d.f.a.c.f.python.JepSingleton - Using JAR connection to /workspace/BOOT-INF/lib/cpg-language-python-5.1.0.jar to extract files into /tmp/cpg_python14439022880182575923

13:18:18.211 [Thread-5] ERROR d.f.aisec.cpg.TranslationManager - An error occurred during parsing of demo.py: Python failed with message: jep.JepException: <class 'ModuleNotFoundError'>: No module named 'de'
13:18:18.218 [Thread-5] INFO d.f.a.cpg.helpers.MeasurementHolder - TranslationManager: Executing Language Frontend done in 426 ms 
```

**Review by akuleshov7 — APPROVED** [review]
I think it works, no?

**Comment by petertrr** [comment]
> I think it works, no?

JEP is working, Python frontend is not (same with Java 11 and 17):
```
5:37:14.279 [Thread-15] INFO d.f.aisec.cpg.TranslationManager - Parsing /tmp/python5272532213918079440/demo.py
05:37:14.311 [Thread-15] INFO d.f.a.c.f.python.JepSingleton - Using JAR connection to /workspace/BOOT-INF/lib/cpg-language-python-5.1.0.jar to extract files into /tmp/cpg_python8763518817481463562

05:37:15.355 [Thread-15] ERROR d.f.aisec.cpg.TranslationManager - An error occurred during parsing of demo.py: Python failed with message: jep.JepException: <class 'ModuleNotFoundError'>: No module named 'de'
05:37:15.355 [Thread-15] INFO d.f.a.cpg.helpers.MeasurementHolder - TranslationManager: Executing Language Frontend done in 1075 ms

```

File: save-cloud-common/build.gradle.kts [hunk 1]
```diff
@@ -17,12 +17,12 @@ kotlin {
     jvm {
         compilations.all {
             kotlinOptions {
-                jvmTarget = Versions.jdk
+                jvmTarget = "11"
```
Comment (akuleshov7) [inline save-cloud-common/build.gradle.kts:20]: revert it back? Dima will check it

**Comment by petertrr** [comment]
> I think it works, no?

Extracted custom builder image build into a workflow to have it prepared. I'll merge it like this now then.

**Pull request merged**